test/unit/vdom/elements.js

```diff
@@ -0,0 +1,34 @@
+import element from '../../lib/element';
+import { symbols, vdom } from '../../../src/index';
+
+const shadowDomV0 = !!document.createElement('div').createShadowRoot;
```

Comment:
What if we just do this in namedslots? `if (!('attachShadow' in document.createElement('div') || 'createShadowRoot' in document.createElement('div')))`